src/anim/controller/anim-blend-tree.js

```diff
@@ -204,6 +94,176 @@ class AnimBlendTree extends AnimNode {
         }
         return count;
     }
+
+    static between(num, a, b, inclusive) {
+        var min = Math.min(a, b),
+            max = Math.max(a, b);
+        return inclusive ? num >= min && num <= max : num > min && num < max;
+    }
+
+    static getAngleRad(a, b) {
+        return Math.atan2(a.x * b.y - a.y * b.x, a.x * b.x + a.y * b.y);
+    }
```

Comment:
if these are generic functions, they could be added to math class

Reply:
I've moved between to the math class and the angle radians function to Vec2 as it operates specifically on them.